### PR TITLE
Changelog 1.22.0

### PR DESCRIPTION
**Pull request by lego963**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+## 1.23.0 (Unreleased)
+
+
+## 1.22.0 (December 03, 2020)
+
+ENHANCEMENTS:
+* **New Data Source:** `opetelekomcloud_dds_instance_v3` ([#725](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/725))
+* `resource/opentelekomcloud_cce_cluster_v3`: Add new argument `authenticating_proxy_ca` ([#727](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/727))
+* `data_source/opentelekomcloud_cce_cluster_v3`: Add new argument `authentication_mode` ([#727](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/727))
+* `resource/opentelekomcloud_obs_bucket`: Add new argument `credentials` for AK/SK pair ([#732](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/732/files))
```
Comment (outcatcher) [inline CHANGELOG.md]: It sounds a bit strange. Maybe change it  to something like
>Allow defining resource-specific AK/SK with `credentials` argument

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,21 @@
...
+* `resource/opentelekomcloud_obs_bucket_object`: Add new argument `credentials` for AK/SK pair ([#732](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/732/files))
+
+BUG FIXES:
+* `resource/opentelekomcloud_identity_credential_v3`: Add the missing documentation ([#731](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/731))
+* `resource/opentelekomcloud_vpnaas_ike_policy_v2`: Fix hardcoded values for `PFS` and `phase1_negotiation_mode` ([#733](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/733))
+* `resource/opentelekomcloud_identity_credential_v3`: Fix changing AK/SK description ([#736](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/736))
```
Comment (outcatcher) [inline CHANGELOG.md]: Mentioning `AK/SK` is redundant here

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,21 @@
-## 1.22.0 (Unreleased)
+## 1.23.0 (Unreleased)
+
+
+## 1.22.0 (December 03, 2020)
+
+ENHANCEMENTS:
```
Comment (anton-sidelnikov) [inline CHANGELOG.md:6]: go.mod approach?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (outcatcher) [reply CHANGELOG.md:6]: It's not a functional change, I don't think is should be in the change log.
When you have binary, you won't see that.

**Review by anton-sidelnikov — DISMISSED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,20 @@
...
+* `data_source/opentelekomcloud_cce_cluster_v3`: Add new argument `authentication_mode` ([#727](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/727))
+* `resource/opentelekomcloud_obs_bucket`: Setting up AK/SK is not required anymore ([#745](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/745))
+* `resource/opentelekomcloud_obs_bucket_object`: Setting up AK/SK is not required anymore ([#745](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/745))
+
+BUG FIXES:
+* `resource/opentelekomcloud_identity_credential_v3`: Add the missing documentation ([#731](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/731))
+* `resource/opentelekomcloud_vpnaas_ike_policy_v2`: Fix hardcoded values for `PFS` and `phase1_negotiation_mode` ([#733](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/733))
+* `resource/opentelekomcloud_identity_credential_v3`: Make `user_id` as Optional ([#737](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/737))
```
Comment (outcatcher) [inline CHANGELOG.md]: `as` is redundant here

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Review by outcatcher — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by outcatcher** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Pull request merged**